contracts/cw721-metadata-onchain/src/lib.rs

```diff
@@ -25,10 +26,38 @@ pub struct Metadata {
     pub youtube_url: Option<String>,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub enum MyQueryMsg {
+    Test,
+}
+
+impl Default for MyQueryMsg {
+    fn default() -> Self {
+        MyQueryMsg::Test
+    }
+}
+
+impl CustomMsg for MyQueryMsg {}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
+pub enum MyExecuteMsg {
+    Test,
+}
+
+impl Default for MyExecuteMsg {
+    fn default() -> Self {
+        MyExecuteMsg::Test
+    }
+}
+
+impl CustomMsg for MyExecuteMsg {}
+
```

Comment:
I like having an example for this, but many people use this contract in production. Feels wrong to have a test Query / Execute message in here.